Find a functionally equivalent code:android.net.wifi.WifiConfiguration.isValid:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * indicates whether the configuration is valid
 * @return true if valid, false otherwise
 * @hide
 */

@@ -1,4 +1,6 @@
 {
+    if (allowedKeyManagement == null)
+        return false;
     if (allowedKeyManagement.cardinality() > 1) {
         if (allowedKeyManagement.cardinality() != 2) {
             return false;

